### PR TITLE
Transl1 fr

### PR DESCRIPTION
**Pull request by paiiou**
Update/Aff french translations for lximage-qt, lxqt-admin & lxqt-config

### CODE REVIEW

**Comment by palinek** [comment]
@paiiou please, correct the auto-test revealed problem:

```
10/52 Test #10: /home/travis/build/lxde/lxqt-l10n/lxqt-config/lxqt-config-appearance .................***Failed    0.06 sec
check_ts(/home/travis/build/lxde/lxqt-l10n/lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts): language(fr_FR) and file name mismatch
```

**Comment by paiiou** [comment]
I corrected, locally, the lxqt-config-file appearance fr.ts.
Then execute  git add. && Git commit - amend -m "Fix lxqt-config-appearance_fr.ts"
Then execute git push --force paiiou transl1_fr: transl1_fr

Should I do another pull request

File: lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr_FR">
```
Comment (palinek) [inline lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts]: The problem still persists...

**Comment by pmattern** [comment]
@paiiou 
In your GitHub account there are two branches `transl1-fr` which fixes the pending problem and `trans1_fr` which doesn't. A push like `git push --force transl1-fr:transl1_fr` is basically  no problem. Didn't look through the content of both branches and hence cannot tell whether this would really make sense, though.
Also, in the snippet [above](https://github.com/lxde/lxqt-l10n/pull/152#issuecomment-254193344) there's a blank between `-` and `amend` which doesn't belong there. Could very well be a copy and paste error but be involved in the missing updated otherwise.

On a side when invoking `git commit --amend` switch `-m` is only needed if you want to change a preexisting commit message but can be omitted otherwise.

**Comment by paiiou** [comment]
Hello Peter

I have actually both branches, of which the corrected branch.
(It is a copy paste error, I have --amend)
Do I request a new pull request from the corrected branch?

(I will soon be able to manage with git !!!)

**Comment by pmattern** [comment]
@paiiou 
Please post the output of `git remote --verbose` and `git branch -vv --all` and tell us which of the two branches is the correct one.

**Comment by paiiou** [comment]
```
$ git remote -v
origin  https://github.com/lxde/lxqt-l10n.git (fetch)
origin  https://github.com/lxde/lxqt-l10n.git (push)
paiiou  https://github.com/paiiou/lxqt-l10n.git (fetch)
paiiou  https://github.com/paiiou/lxqt-l10n.git (push)
```

```
$ git branch -vv --all
  master                              5fb50da [origin/master] Update lxqt-session_nl.ts
* transl1_fr                          8101e77 [paiiou/transl1_fr : en avance de 1, en retard de 1] Fix lxqt-config-appearance_fr.ts
  remotes/origin/HEAD                 -> origin/master
  remotes/origin/authors-wip          2bc90fc Create AUTHORS in subdirs
  remotes/origin/master               5fb50da Update lxqt-session_nl.ts
  remotes/origin/mst_pt               d60ae4c Update Portugese translations
  remotes/origin/releases-0.11.x      ed7ab45 Release 0.11.0: Add changelog
  remotes/origin/use-lxqt-build-tools 1313ce3 Use lxqt-build-tools
  remotes/paiiou/master               5fb50da Update lxqt-session_nl.ts
  remotes/paiiou/transl1-fr           8101e77 Fix lxqt-config-appearance_fr.ts
  remotes/paiiou/transl1_fr           aeb73e9 Update lximage-qt, lxqt-admin, lxqt-config french translations
```

Voilà

File: lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts (hunk 1 above)
Comment (paiiou) [reply lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts]: I just looked on 'github paiiou' the corrected branch and read 8101e77
Can this information help?

**Comment by paiiou** [comment]
As I am very curious, where did I wrong?

**Comment by pmattern** [comment]
Look, meanwhile there are three commits involved.

| commit | branch(es) | commit message | files addressed |
| --- | --- | --- | --- |
| 67b8e12 | transl1_fr, transl1-fr | Update lximage-qt, lxqt-admin, lxqt-config french translations | lxqt-config_fr.ts only |
| aeb73e9 | transl1_fr | (same) | 11 files |
| 8101e77 | transl1-fr | Fix lxqt-config-appearance_fr.ts | 11 files (at first glance the same as in aeb73e9) |

So apparently something went wrong regarding commit messages and those two branches. I haven't got time to look through all those files but it can be assumed that the two commits in branch `transl1-fr` are what's actually needed.
So you may try to push it into this PR as follows: Create a so-called tracking branch that's linked with branch `transl1-fr` in your account

```
git checkout -b transl1-fr paiiou/transl1-fr
```

Force-push from this tracking branch to branch `transl1_fr` in your account as this is the one corresponding with this PR.

```
git push --force paiiou transl1-fr:transl1_fr
```

**Comment by palinek** [comment]
> As I am very curious, where did I wrong?

Your local `transl1_fr` branch isn't synchronized with the remote one:

```
* transl1_fr                          8101e77 [paiiou/transl1_fr : en avance de 1, en retard de 1] Fix lxqt-config-appearance_fr.ts
...
  remotes/paiiou/transl1_fr           aeb73e9 Update lximage-qt, lxqt-admin, lxqt-config french translations
```

... so basicaly you just need to force-push: `git push --force paiiou transl1_fr:transl1_fr`

**Comment by paiiou** [comment]
Thank you both.
bash stores the different commands that I have made.
So, I was able to reconstruct the course and find errors:
1) a commit then I was not in the root directory
2) another commit, with the same commentary from the root directory
3) a push towards my fork in GitHub: -u git push paiiou transl1_fr
=> Far, it worked, but a file was incorrect (en_US instead fr)

Fixed file
And there is a mistake: git push --force paiiou transl1_fr: transl1-fr

I'll run good command Thank you both.
bash stores the different commands that I have made.
I was able to reconstruct the course and find errors:
1) a commit then I was not in the root directory
2) another commit, with the same commentary from the root directory
3) a push towards my fork in GitHub: -u git push paiiou transl1_fr
=> Far, it worked, but a file was incorrect (fr_FR instead fr)

Fixed file
And there is a mistake: git push --force paiiou transl1_fr: transl1-fr

I'll run good command: git push --force paiiou transl1_fr: transl1_fr

**Comment by palinek** [comment]
> I'll run good command: git push --force paiiou transl1_fr: transl1_fr

Did you push it already? It's not here...

**Comment by paiiou** [comment]
It is now done

**Comment by paiiou** [comment]
Phew !!! But I learned a lot. thanks again

**Comment by palinek** [comment]
landed in @51c802a, thanks

**Pull request closed without merging**